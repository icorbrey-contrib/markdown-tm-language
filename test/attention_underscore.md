Left letter:

ax_xb

ax_ b

ax_.b

Left whitespace:

a _xb

a _ b

a _.b

Left punc:

a._xb

a._ b

a._.b
