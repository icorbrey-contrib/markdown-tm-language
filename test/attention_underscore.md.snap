>Left letter:
#^^^^^^^^^^^^^ text.md meta.paragraph.md
>
>ax_xb
#^^^^^^ text.md meta.paragraph.md
>
>ax_ b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>ax_.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>Left whitespace:
#^^^^^^^^^^^^^^^^^ text.md meta.paragraph.md
>
>a _xb
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>a _ b
#^^^^^^ text.md meta.paragraph.md
>
>a _.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>Left punc:
#^^^^^^^^^^^ text.md meta.paragraph.md
>
>a._xb
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>a._ b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>
>a._.b
#^^ text.md meta.paragraph.md
#  ^ text.md meta.paragraph.md string.other.emphasis.underscore.md
#   ^^^ text.md meta.paragraph.md
>